### PR TITLE
Fix typos and wrong pip version range

### PR DESCRIPTION
**Pull request by kinow**

### CODE REVIEW

File: cwl_utils/cwl_expression_refactor.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def __call__(
 
 
 def arg_parser() -> argparse.ArgumentParser:
-    """Build the arrgument parser."""
+    """Build the argument parser."""
```
Comment (kinow) [inline cwl_utils/cwl_expression_refactor.py:62]: :pirate_flag:

File: cwl_utils/cwl_v1_0_expression_refactor.py [hunk 2]
```diff
@@ -216,7 +216,7 @@ def traverse(
     skip_command_line1: bool,
     skip_command_line2: bool,
 ) -> Tuple[Union[cwl.CommandLineTool, cwl.ExpressionTool, cwl.Workflow], bool]:
-    """Convert the given process and any subprocesess."""
```
Comment (kinow) [inline cwl_utils/cwl_v1_0_expression_refactor.py:219]: There were typos in the cwl files under the parser package, but there's a note at the top of those files saying they are auto-generated (i.e. better fix the typos at the source, if not already fixed). I noticed these other *expression_refactor files have similar typos, but not sure if these are also auto-generated? Happy to remove them from the PR if needed :+1:

File: docs/conf.py [hunk 3]
```diff
@@ -18,8 +18,8 @@
 # -- Project information -----------------------------------------------------
 
 project = 'cwl-utils'
-copyright = '2019, CWL Commmunity'
-author = 'CWL Commmunity'
+copyright = '2019, CWL Community'
+author = 'CWL Community'
```
Comment (kinow) [inline docs/conf.py:22]: This is what started this PR. Went to look at the Sphinx issue in #146 , and noticed the typo above. Then ran the Code Inspection in PyCharm to check for other typos.

File: cwl_utils/cwl_v1_0_expression_refactor.py (hunk 2 above)
Comment (mr-c) [reply cwl_utils/cwl_v1_0_expression_refactor.py:219]: The expression refactor files are not autogenerated. So typo fixes are appreciated, thanks!

**Pull request merged**